src/styles.ts

```diff
@@ -162,6 +162,8 @@ export const commonArticleStyles = ({ kicker }: PillarStyles): SerializedStyles
         a {
             text-decoration: none;
         }
+
+        ${darkModeCss`background: black;`}
```

Comment:
Do we not have a palette colour for this?

Reply:
I couldn't see one, but I thought we did too..
Not in source-components either:

```
declare const palette: {
    brand: {
        dark: string;
        main: string;
        bright: string;
        pastel: string;
        faded: string;
    };
    yellow: {
        dark: string;
        main: string;
    };
    neutral: {
        7: string;
        20: string;
        46: string;
        60: string;
        86: string;
        93: string;
        97: string;
        100: string;
        specialReport: string;
        darkMode: string;
    };
    error: {
        main: string;
        bright: string;
    };
    news: {
        dark: string;
        main: string;
        bright: string;
        pastel: string;
        faded: string;
    };
    opinion: {
        dark: string;
        main: string;
        bright: string;
        pastel: string;
        faded: string;
    };
    sport: {
        dark: string;
        main: string;
        bright: string;
        pastel: string;
        faded: string;
    };
    culture: {
        dark: string;
        main: string;
        bright: string;
        pastel: string;
        faded: string;
    };
    lifestyle: {
        dark: string;
        main: string;
        bright: string;
        pastel: string;
        faded: string;
    };
    labs: {
        dark: string;
        main: string;
    };
};
export { palette };
```